nerodia/keys.py

```diff
@@ -0,0 +1,3 @@
+import selenium
+
+Keys = selenium.webdriver.common.keys.Keys
```

Comment:
This could all be replaced with one line of

`from selenium.webdriver.common.keys import Keys`  with a #noqa so flake8 doesn't get mad

Reply:
I like this way better, particularly since it doesn't introduce any weird Python 2 vs 3 issues to test